erizo_controller/erizoJS/models/Connection.js

```diff
@@ -38,12 +38,12 @@ class Connection extends events.EventEmitter {
     this.mediaConfiguration = 'default';
     //  {id: stream}
     this.mediaStreams = new Map();
+    this.options = options;
+    this.metadata = this.options.metadata || {};
```

Comment:
Do we still need this? I've seen this PR uses `this.options.metadata` in general